### PR TITLE
Adopt erc standards

### PR DESCRIPTION
**Pull request by darrenvechain**
This PR focuses on updating several VeChain Improvement Proposals (VIPs) to recommend the adoption of corresponding Ethereum Request for Comment (ERC) standards for improved interoperability.

Detailed summary:
VIP-210: Rejected in favor of ERC1155 standard, migration strongly discouraged.
VIP-190: Updated wording to refer to VechainThor instead of VeChain.
VIP-181: Withdrawn in favor of ERC721 standard, migration strongly discouraged.
VIP-180: Superseded by ERC20 standard, migration strongly discouraged.
Added new VIP proposing the adoption of ERC standards for VeChain contracts.

### CODE REVIEW

**Review by nwbrettski — APPROVED** [review]
LGTM

File: vips/VIP-240.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+Title: ERC Contract Adoption
+Description: |
+  This VeChain Improvement Proposal (VIP) advocates for the adoption of Ethereum Request for Comment (ERC) standards, aiming to eliminate duplication and enhance interoperability by aligning VeChain contracts with their Ethereum counterparts.
```
Comment (libotony) [inline vips/VIP-240.md:4]:
```suggestion
  This Vechain Improvement Proposal (VIP) advocates for the adoption of Ethereum Request for Comment (ERC) standards, aiming to eliminate duplication and enhance interoperability by aligning Vechain contracts with their Ethereum counterparts.
```
I think there are more `VeChain`

File: vips/VIP-240.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Specification
+
+To promote alignment with widely accepted Ethereum standards and enhance interoperability, the following actions are recommended:
+
+1. **Supersede VIP180 in favor of ERC20:**
+   - Developers are strongly encouraged to migrate from VIP180 to the ERC20 standard for fungible token implementations on the VechainThor Network.
+   - New projects intending to create fungible tokens should prioritize the use of ERC20 rather than VIP180.
+
+2. **Withdraw VIP181 in favor of ERC721:**
```
Comment (libotony) [inline vips/VIP-240.md:34]:
```suggestion
2. **Supersede VIP181 in favor of ERC721:**
```

File: vips/VIP-240.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+   - Developers should discontinue the use of VIP181 for non-fungible token (NFT) implementations and adopt the ERC721 standard for improved compatibility and industry-wide conformity.
+   - New NFT projects are advised to use ERC721 instead of VIP181.
+
+3. **Reject VIP210 in favor of ERC1155:**
```
Comment (libotony) [inline vips/VIP-240.md:38]:
```suggestion
3. **Supersede VIP210 in favor of ERC1155:**
```

**Comment by darrenvechain** [comment]
@libotony pushed the updates for zh_CN. Could you take a look?

File: the-status-page.md [hunk 4]
```diff
@@ -42,4 +40,11 @@ All Vechain Improvement Proposals
 ## Superseded
 | No.                      | Title                          | Owner   | Category    | Status | Reviewers |
 | ------------------------ | ------------------------------ | ------- | ----------- | ------ | ---- |
+| [180](./vips/VIP-180.md) | Fungible Token Standard        | Vechain | Application | Final  | - |
```
Comment (libotony) [inline the-status-page.md:43]:
```suggestion
| [180](./vips/VIP-180.md) | Fungible Token Standard        | Vechain | Application | Superseded  | - |
```

File: the-status-page.md [hunk 5]
```diff
@@ -42,4 +40,11 @@ All Vechain Improvement Proposals
...
+| [181](./vips/VIP-181.md) | Non-Fungible Token Standard    | Vechain | Application | Accepted  | - |
```
Comment (libotony) [inline the-status-page.md:44]:
```suggestion
| [181](./vips/VIP-181.md) | Non-Fungible Token Standard    | Vechain | Application | Superseded  | - |
```

File: the-status-page.md [hunk 6]
```diff
@@ -42,4 +40,11 @@ All Vechain Improvement Proposals
...
+| [210](./vips/VIP-210.md) | Semi-Fungible Token(SFT) Standard | Vechain | Application | Draft | - |
```
Comment (libotony) [inline the-status-page.md:45]:
```suggestion
| [210](./vips/VIP-210.md) | Semi-Fungible Token(SFT) Standard | Vechain | Application | Superseded | - |
```

File: vips/VIP-180-zh_CN.md [hunk 7]
```diff
@@ -2,11 +2,17 @@
...
+Author: Vechain Foundation
 Status: Final
 CreatedAt: 2018-09-01
 ---
 
+## ⚠️ 取代 ⚠️
+
+**此标准 (VIP-180) 已被以太坊上的 [ERC20](https://github.com/ethereum/ERCs/blob/master/ERCS/erc-20.md) 标准取代。强烈建议不要开发和使用 VIP-180，建议现有实现迁移到 ERC20。**
+
+有关在 VechainThor 网络上使用 ERC20 作为可替代代币的详细指南，请参阅文档[此处](https://github.com/ethereum/ERCs/blob/master/ERCS/erc-20.md)。
```
Comment (libotony) [inline vips/VIP-180-zh_CN.md:14]:
```suggestion
有关在 VechainThor 网络上使用 ERC20 作为同质化代币的详细指南，请参阅文档[此处](https://github.com/ethereum/ERCs/blob/master/ERCS/erc-20.md)。
```

File: vips/VIP-180-zh_CN.md [hunk 8]
```diff
@@ -2,11 +2,17 @@
 VIP: 180
 Title: Fungible Token Standard
 Category: Application
-Author: VeChain Foundation
+Author: Vechain Foundation
 Status: Final
```
Comment (libotony) [inline vips/VIP-180-zh_CN.md:6]:
```suggestion
Status: Superseded
```

File: vips/VIP-181-zh_CN.md [hunk 9]
```diff
@@ -2,11 +2,17 @@
 VIP: 181
 Title: Non-fungible Token Standard
 Category: Application
-Author: VeChain Foundation
+Author: Vechain Foundation
 Status: Accepted
```
Comment (libotony) [inline vips/VIP-181-zh_CN.md:6]:
```suggestion
Status: Superseded
```

**Comment by libotony** [comment]
> @libotony pushed the updates for zh_CN. Could you take a look?

I have made some suggestions, other part looks good to me.

**Review by libotony — APPROVED** [review]

**Comment by NecoSherry** [comment]
Hi @darrenvechain, each VIP requires a minimum of three reviewers. Could you please reopen this PR and request an additional reviewer to approve it? Also, we have specific rules for assigning numbers, and the number you added is not comply with them. We will assign the correct number when merging again.

**Review by daithihearn — COMMENTED** [review]
LGTM

**Pull request merged**